### PR TITLE
OCM-8776 | test: Automated case id:35420，74399 Create cluster with channel group

### PR DESCRIPTION
**Pull request by jameszwang**

### CODE REVIEW

File: tests/e2e/test_rosacli_cluster_post.go [hunk 1]
```diff
@@ -237,3 +238,52 @@ var _ = Describe("Healthy check",
...
+			func() {
+				clusterName := "ocp-35420"
+				profile := profilehandler.LoadProfileYamlFileByENV()
+
+				By("Check if the cluster using right channel group")
+				versionOutput, err := clusterService.GetClusterVersion(clusterID)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(versionOutput.ChannelGroup).To(Equal(profile.ChannelGroup))
+
+				By("Create cluster with version not in channel group")
```
Comment (xueli181114) [inline tests/e2e/test_rosacli_cluster_post.go:282]: Split the negative checkpoint to another case with day1-negative labeled

File: tests/e2e/test_rosacli_cluster_post.go [hunk 2]
```diff
@@ -237,3 +238,52 @@ var _ = Describe("Healthy check",
...
+			clusterService = rosaClient.Cluster
+		})
+
+		AfterEach(func() {
+			By("Clean remaining resources")
+			rosaClient.CleanResources(clusterID)
+		})
+
+		It("User can create cluster with channel group - [id:35420]",
+			labels.Medium, labels.Runtime.Day1Post,
```
Comment (xueli181114) [inline tests/e2e/test_rosacli_cluster_post.go:272]: day1-post should be Critical importance

File: tests/e2e/test_rosacli_cluster_post.go (hunk 1 above)
Comment (jameszwang) [reply tests/e2e/test_rosacli_cluster_post.go]: create a new test cases ocp-74399 and split the checkpoint to the new case with day1-negtaive label.

File: tests/e2e/test_rosacli_cluster_post.go (hunk 2 above)
Comment (jameszwang) [reply tests/e2e/test_rosacli_cluster_post.go]: updated the label to critical.

**Pull request merged**